github_pr.py

```diff
@@ -108,12 +109,64 @@ def github_list_prs(**args):
     elif 'head' in args and args['head']:
         prs = _load_prs_by_branch(**args)
         _print_prs(prs, **args)
+    elif 'filter' in args and args['filter']:
+        prs = github_filter_prs(**args)
+        #_print_prs(prs, **args)
     else:
         prs = repo.get_pulls()
         _print_prs(prs, **args)
     return list_return_obj
 
 
+def github_filter_prs(**args):
+    """
+    Filters prs to return only what is contained in the filters
+    """
+    gh = Github(args['token'])
+    repo = gh.get_repo(args['repo'])
+    all_prs = [{'pr':pr} for pr in repo.get_pulls()]
```

Comment:
i find all the swapping / setting of `pr_issue` and `all_prs` to be kinda confusing. is `all_prs` pr objects or the dictionaries that `pr_issue` is filled with?